pygmy/model/link.py

```diff
@@ -1,17 +1,21 @@
 import time
 import binascii
 import datetime
+from base64 import b64encode
+from io import BytesIO
+import qrcode
 
 from sqlalchemy import event, and_, or_, DDL
 from sqlalchemy.exc import IntegrityError
 from sqlalchemy.orm import relationship
 from sqlalchemy import (Column, String, Integer, Boolean,
-                        BigInteger, Unicode, DateTime)
+                        BigInteger, Unicode, DateTime, VARCHAR)
 from urllib.parse import urlparse
 
 from pygmy.database.base import Model
 from pygmy.database.dbutil import dbconnection, utcnow
 from pygmy.exception.error import ShortURLUnavailable
+from pygmy.exception.error import QrCodeGenerationFailed
```

Comment:
Could be merged into the line above `from pygmy.exception.error import ShortURLUnavailable, QrCodeGenerationFailed`